x-pack/plugins/code/server/lsp/request_expander.ts

```diff
@@ -191,8 +191,8 @@ export class RequestExpander implements ILanguageServerHandler {
 
         if (timeout > 0 && ws.initPromise) {
           try {
-            const elasped = Date.now() - startTime;
-            await promiseTimeout(timeout - elasped, ws.initPromise);
```

Comment:
From test:
`timeoutForInitializeMs = 100`
`Date.now() - startTime` should >= 0
So `timeout -elapsed should <= 100`

But initialize cost 200ms to finish, so an `InitializingError` should throw here.